src/app/zap-templates/zcl/data-model/silabs/general.xml

```diff
@@ -586,7 +586,7 @@ limitations under the License.
       <arg name="status" type="Status"/>
       <arg name="alarmCode" type="ENUM8"/>
       <arg name="clusterId" type="CLUSTER_ID"/>
-      <arg name="timeStamp" type="INT32U"/>
+      <arg name="timeStamp" type="UTC"/>
```

Comment:
Can this and other UTC bits just be `epoch-s`?  If not, why not?

Reply:
Done， thanks